build/_k-test.shade

```diff
@@ -58,7 +59,10 @@ projectFile=''
                     testArgs = " -parallel none";
                 }
 
-                Dnx("test" + testArgs, projectFolder);
+                if (!IsLinux || string.IsNullOrEmpty(KOREBUILD_TEST_SKIPMONO))
```

Comment:
Sorry, I'm not sure if I understand the logic inside `if`.  When do we execute this command and why the difference between Linux and OS X?